linked_list/linked_list.py

```diff
@@ -6,29 +6,42 @@ def __init__(self):
         """
         Initialize the LinkedList here
         """
-        pass
+        self.head = None
+        self.tail = self.head
 
     def get_head(self):
-        pass
+        return self.head
 
     def get_tail(self):
-        pass
+        return self.tail
 
     def add(self, value):
         """
         Adds the value to the end of the list
 
         :param value: the value to be added
         """
-        pass
+        new_node = Node(value)
+        if self.head is None:
+            self.head = new_node
+            self.tail = self.head
+        else:
+            previous_node = self.head
+            while previous_node.next:
```

Comment:
You can add the new node directly to the tail instead of using a loop here

Reply:
done done